web/vite.config.ts

```diff
@@ -10,6 +10,20 @@ export default defineConfig(() => ({
     svelte(),
     nodeResolve({
       extensions: ['.js', '.ts']
-    })
+    }),
+    {
+      enforce: 'pre' as const,
+      // Deploy section.
```

Comment:
```suggestion
```

Let’s remove these comments. In this project we prefer to have it only for complex cases.